roles/container_runtime/defaults/main.yml

```diff
@@ -91,7 +91,3 @@ pause_image: "{{ l_os_registry_url | regex_replace('${component}' | regex_escape
 l_required_docker_version: '1.13'
 
 l_crio_var_sock: "/var/run/crio/crio.sock"
-
-l_insecure_crio_registries: "{{ '\"{}\"'.format('\", \"'.join(l2_docker_insecure_registries)) }}"
-l_crio_registries: "{{ l2_docker_additional_registries + ['docker.io'] }}"
-l_additional_crio_registries: "{{ '\"{}\"'.format('\", \"'.join(l_crio_registries)) }}"
```

Comment:
all these 3 must be set in `/etc/containers/registries.conf` at openshift installation or it will fail (the change is fine overall, but we must set these registries somewhere :))

Reply:
From what I understand, `l_insecure_crio_registries` is a formatted version of `l2_docker_insecure_registries`, and `l2_docker_insecure_registries` is being set in `/etc/containers/registries.conf` https://github.com/openshift/openshift-ansible/blob/master/roles/container_runtime/templates/registries.conf#L31. Since we are no longer using this in `crio.conf` we can remove them and just use the `l2_docker_...` variables.

Reply:
just triple checking, so inseucre_registries and additional_registries will still be added to registries.conf if I set them in the ansible inventory? (again, I just want to make sure we don't break installation)

Reply:
> So l2_docker_insecure_registries picks up the registries from openshift_docker_insecure_registries and same for the additional_registries. So when you set the openshift additional and insecure registries, the registries.conf file should pick it up (if this what you were asking).

yeah, alright, so everything that was applied to crio.conf in terms of *_registries is still applied to registries.conf, so removing crio.conf tweaks still leave us with reigistries.conf correctly populated.

Reply:
Yup! CRI-O is still getting all the registries in registries.conf :)